HW_3/task_4.py

```diff
@@ -0,0 +1,28 @@
+"""
+Написать программу по переводу целого числа из десятичной системы счисления в двоичную.
+
+Ввод: значение типа <int>
+Вывод: значение типа <int>
+
+Примеры:
+45
+101101
+
+3
+11
+
+2
+10
+"""
+number = int(input("Enter the number: "))
+def binary(number):
+    if number == 0:
+        return 0
+    
+    remains = binary(number // 2)
+    number = number % 2 + 10 * remains
+#    print(number)
+    return number
```

Comment:
Впервые вижу, чтобы решали, не используя строки. Очень круто.
```suggestion
    return number % 2 + 10 * binary(number // 2)
```